### ISSUE TITLE
Real desktop app or persistent PWA

### ISSUE THREAD

**Issue by Fabio53443** [ente-io/auth#243]
Title, basically. 
The current desktop experience is subpar to what other competitors, like Authy offer. The webapp is slow to log in (possibly because it has to decode the OTPs etc) and it's not available offline (or at least not on firefox).
It would be great if a real desktop client was made, especially since the android/ios app is built on flutter so it wouldn't be a huge effort. 
Thanks!

**Comment by lbr-dev** [comment]
I agree or maybe some kind of Firefox browser extension like password managers have

**Comment by Xyz00777** [comment]
an desktop app would be mutch better because its easier to attack an browser with sutch extension in comparison to an desktop app. Ans please dont forget linux if you do an windows app :) these would be awesome!

**Comment by Syraphic** [comment]
https://github.com/ente-io/auth/releases/tag/nightly
